docs/quickstarts/nextjs.mdx

```diff
@@ -118,68 +120,113 @@ The [`<ClerkProvider>`](/docs/components/clerk-provider) component provides acti
 
-Clerk offers a set of [prebuilt components](/docs/components/overview) to add functionality to your app with minimal effort. The [`<UserButton />`](/docs/components/user/user-button) allows users to manage their account information and log out, completing the full authentication circle.
+<CodeBlockTabs type="router" options={["App Router", "Pages Router"]}>
+```tsx filename="app/page.tsx" {1,6-11}
+import { SignedIn, SignedOut } from "@clerk/nextjs";
 
-Add the `<UserButton />` anywhere inside `<ClerkProvider>` in your app. First, add `import { UserButton } from "@clerk/nextjs";` to the top of your file. Then, add `<UserButton afterSignOutUrl="/"/>`. The `afterSignOutUrl` prop lets you customize what page the user will be redirected to after sign out.
+export default function Home() {
+  return (
+    <div className="h-screen">
+      <SignedOut>
+        <p>This content is public. Only signed out users can see this.</p>
+      </SignedOut>
+      <SignedIn>
+        <p>This content is private. Only signed in users can see this.</p>
+      </SignedIn>
+    </div>
+  )
+}
+```
+
+```tsx filename="pages/index.tsx" {1,6-11}
+import { SignedIn, SignedOut } from "@clerk/nextjs";
+
+export default function Home() {
+  return (
+    <>
+      <SignedOut>
+        <p>This content is public. Only signed out users can see this.</p>
+      </SignedOut>
+      <SignedIn>
+        <p>This content is private. Only signed in users can see this.</p>
+      </SignedIn>
+    </>
+  );
+}
+``` 
+</CodeBlockTabs>
+
+Try accessing your app now by visiting [`http://localhost:3000`](http://localhost:3000) to see what signed out users will see. Next you will add a button so you can sign in.
+
+### Embed the `<UserButton />` and `<SignInButton />`
+
+Clerk offers a set of [prebuilt components](/docs/components/overview) to add functionality to your app with minimal effort. The [`<UserButton />`](/docs/components/user/user-button) allows users to manage their account information and log out, completing the full authentication circle. The [`<SignInButton />`(/docs/components/unstyled/sign-in-button)] is an unstyled component designed to wrap about your button. It can also act as a basic button on its own.
+
+Add the `<UserButton />` and `<SignInButton />` components. First, add `import { UserButton, SignedInButton } from "@clerk/nextjs";` to the top of your file. Then, add `<UserButton afterSignOutUrl="/"/>` inside of the `<SignedIn />` components. The `afterSignOutUrl` prop lets you customize what page the user will be redirected to after sign out. Lastly, add `<SignInButton />` comoponent inside of the `<SignedOut /> components`

```

Comment:
Maybe this can be restructured & simplified a bit, I haven't thought of a good way myself yet.